packages/evals/benchmark_memory_module.py

```diff
@@ -90,9 +90,9 @@ def create_message(**kwargs: Any) -> Union[AssistantMessage, UserMessage]:
 
 
 def run_benchmark(
-    name: str,
+    name: Optional[str],
```

Comment:
In python, is it ok for optional to be mixed with required? like in TS required params need to go before optional.

Reply:
Hmm, good question. It doesn't really matter much for benchmark here. This doesn't get called directly.